### PR TITLE
Add closing roadmap 2020 review (June 2020)

### PR DESCRIPTION
**Pull request by adityasaky**
This adds the final roadmap review for our 2020 roadmap.

### CODE REVIEW

File: roadmap-reviews/2020/review_3_june_20.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+## Cross-implementation interoperability
+
+- __in-toto-golang__ Apart from
+  [ed25519 signing](https://github.com/in-toto/in-toto-golang/pull/48) and
+  [bug fixes in the canonical JSON implementation](https://github.com/in-toto/in-toto-golang/pull/50)
+  discussed in the previous reviews, work began in earnest on bringing
+  in-toto's _runlib_ to the Go implementation. Christian Rebischke (@shibumi)
+  joined the in-toto team to implement this as part of Google Summer of Code
+  (GSoC). in-toto is participating in GSoC through the Cloud Native Computing
+  Foundation(CNCF). In the process, Christian also identified a nil pointer
```
Comment (lukpueh) [inline roadmap-reviews/2020/review_3_june_20.md:15]:
```suggestion
  Foundation (CNCF). In the process, Christian also identified a nil pointer
```

File: roadmap-reviews/2020/review_3_june_20.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+  dereference in Go's standard library for RSA, and he opened a
+  [pull request](https://go-review.googlesource.com/c/go/+/240008/) to address
+  it. Efforts are also underway to improve interoperability between this
+  implementation and the reference implementation.
```
Comment (lukpueh) [inline roadmap-reviews/2020/review_3_june_20.md:19]: Not sure how much detail we want to provide, but @shibumi also added symlink recording support and works on exclude patterns:  https://github.com/in-toto/in-toto-golang/pulls?q=is%3Apr+author%3Ashibumi+

File: roadmap-reviews/2020/review_3_june_20.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+- __in-toto-rs__ Santiago Torres-Arias (@SantiagoTorres) began work on a Rust
+  implementation of in-toto. This broadens in-toto's attestation generation
+  capabilities, and in-toto-rs is being used to generate attestations in
+  _rebuilderd_ as part of the Reproducible Builds effort.
+
+- __in-toto-java__ The Java implementation saw relatively lesser activity this
+  year. The only activity was the updating of some documentation and a minor
+  update to add a new feature to the in-toto Jenkins plugin. Additionally, a
+  third-party SaaS provider is working on a Java framework based on in-toto.
```
Comment (lukpueh) [inline roadmap-reviews/2020/review_3_june_20.md:29]: Add links?

File: roadmap-reviews/2020/review_3_june_20.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+  Foundation(CNCF). In the process, Christian also identified a nil pointer
+  dereference in Go's standard library for RSA, and he opened a
+  [pull request](https://go-review.googlesource.com/c/go/+/240008/) to address
+  it. Efforts are also underway to improve interoperability between this
+  implementation and the reference implementation.
+
+- __in-toto-rs__ Santiago Torres-Arias (@SantiagoTorres) began work on a Rust
+  implementation of in-toto. This broadens in-toto's attestation generation
+  capabilities, and in-toto-rs is being used to generate attestations in
+  _rebuilderd_ as part of the Reproducible Builds effort.
```
Comment (lukpueh) [inline roadmap-reviews/2020/review_3_june_20.md:24]: Add links?

File: roadmap-reviews/2020/review_3_june_20.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+## Maturing of existing sub-projects
+
+As noted in our roadmap last year, we intended to mature the sub-projects that
+have sprouted, as well as beef up the in-toto integration in third-party
+projects.
+
+### Grafeas integration
+
+Grafeas is an open source artifact metadata API developed by Google. We worked
+with the Grafeas team to
```
Comment (lukpueh) [inline roadmap-reviews/2020/review_3_june_20.md:76]: The leading force "Aditya Sirish (@adityasaky)" should be mentioned here.

File: roadmap-reviews/2020/review_3_june_20.md [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+
+## Community Meetings
+
+In an effort to interact directly with stakeholders, both to update them as
+well as understand their needs and use cases, we have begun conducting regular
+community meetings. This has helped grow the in-toto community which is also
+good for software supply chain security as a whole. If you're interested in
+joining the next community meeting, please subscribe to the
+[in-toto mailing list](https://groups.google.com/forum/#!forum/in-toto-public).
+You can also find notes on the meetings held thus far.
```
Comment (lukpueh) [inline roadmap-reviews/2020/review_3_june_20.md:167]: Where?

File: roadmap-reviews/2020/review_3_june_20.md [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+
+## Closing Remarks
+
+That brings us to the end of the in-toto roadmap for 2020. We have a robust
+community that's working in many different projects and ecosystems to improve
+software supply chain security. We have met our goals of formalizing an
+approach to integrate TUF and in-toto to provide end-to-end guarantees, as well
+as a way to extend in-toto's properties to supply chain artifacts beyond files,
+through ITE-2, ITE-3, and ITE-4. We have also made progress towards achieving
+spec complaince in our reference implementation and in making in-toto-golang
```
Comment (lukpueh) [inline roadmap-reviews/2020/review_3_june_20.md:177]:
```suggestion
spec compliance in our reference implementation and in making in-toto-golang
```

File: roadmap-reviews/2020/review_3_june_20.md [hunk 8]
```diff
@@ -0,0 +1,183 @@
...
+fully featured to enable its integration in cloud nativ / cloud first contexts.
```
Comment (lukpueh) [inline roadmap-reviews/2020/review_3_june_20.md:178]:
```suggestion
fully featured to enable its integration in cloud native / cloud first contexts.
```

**Comment by adityasaky** [comment]
Thanks for the review, @lukpueh! I think I addressed them. I also included a mention of our acceptance into the CNCF sandbox which I'd missed before.

**Comment by adityasaky** [comment]
Pre-merge note: I'd prefer to link to `in-toto-rs` on the in-toto org if we're moving it.

**Pull request merged**